### ISSUE TITLE
Schema, Protocol, or Scheme

### ISSUE THREAD

**Issue by yiufung** [zweifisch/ob-http#38]
I wish to submit request to https website with `:host` option, but it's default to `http`, so I figured how to change it. Looking at the source code, found that it's called `schema` here. Changed and works fine. 

It seems a bit confusing here: `curl` manual (7.61.1) mainly uses `protocol`, and mentions a bit about `scheme`, but not `schema`. Shall we change it to a clearer term, or add docs for it?

**Comment by ewestbrook** [comment]
You're right.  Per [RFC 3986](https://tools.ietf.org/html/rfc3986), the correct term is "scheme".

**Comment by ewestbrook** [comment]
PR #39 submitted.
